content/docs/examples/userauthentication.md

```diff
@@ -4,28 +4,56 @@
     "title": "Integrate User Authentication",
     "linktitle": "userauthentication",
     "card_icon": "ti-id-badge",
-    "card_body": "Learn how add User Authentication with Auth0",
+    "card_body": "Learn how to add User Authentication with Auth0",
     "weight": "2"
 }
 
 
 # Add User Authentication to Your Application
 {{<header_divider>}}
 
-User Authentication and Management is one of the most important aspects aspects in application development.
+User Authentication and Management is one of the most important aspects in application development.
 
 With the `Cdev API` resource, we can add granular [User Authorization](https://auth0.com/intro-to-iam/what-is-authorization/) and [Role-Based Access Control (RBAC)](https://auth0.com/docs/manage-users/access-control/rbac) to any backend. The `Cdev API` resource supports
-user authentication through JWT Tokens (need more details on the specifics).
+user authentication through [JSON Web Tokens(JWT)](https://jwt.io/introduction).
 
-For these examples, we will be using [Auth0](https://auth0.com/) as our User Authorization platform and integrating their service into our API's. Auth0 provides great tutorials and we based the following examples off their [official documentation](https://auth0.com/blog/securing-aws-http-apis-with-jwt-authorizers/) and a [helpful blog by Sandrino Di Mattia](https://sandrino.dev/blog/aws-api-gateway-jwt-auth0)
+{{<tool_tip key="info" summary="JWT Authentication">}}
+You can learn more about JWT authentication [here](https://auth0.com/docs/secure/tokens/json-web-tokens).
+{{</tool_tip>}}
+
+For these examples, we will be using [Auth0](https://auth0.com/) as our User Authorization platform and integrating their service into our API's. Auth0 provides great tutorials and we based the following examples off their [official documentation](https://auth0.com/blog/securing-aws-http-apis-with-jwt-authorizers/) and a [helpful blog by Sandrino Di Mattia](https://sandrino.dev/blog/aws-api-gateway-jwt-auth0).
 
 {{<break 1>}}
-## Create a Basic API
-Use the `user-auth` template as a starting place 
-```bash
+
+## Set Up
+You will need to use the **[Getting Started](/docs/gettingstarted)** section and follow the instructions for installing the **[Cdev SDK](/docs/gettingstarted/installingcdev)** with the following **EXCEPTION**:
+
+Initialize the project with the `user_auth` template instead of the `quick-start` template by using the following command, 
+
+```bash 
 cdev init user-auth-demo --template user-auth
 ```
-You should have a `src/resources.py` file that looks like 
+instead of this command:
+```bash 
```

Comment:
I do not think we need this section with the `quick-start` command as what not to do. Having the above lines and tooltip is sufficient.